src/connectedmachine/setup.py

```diff
@@ -10,7 +10,7 @@
 from setuptools import setup, find_packages
 
 # HISTORY.rst entry.
-VERSION = '1.0.0b2'
+VERSION = '1.0.0'
 try:
```

Comment:
if stable version is expected, azext.isPreview: true in azext_metadata.json should be removed